dynamic-plugins.default.yaml

```diff
@@ -426,6 +426,33 @@ plugins:
                     - isKind: resource
                     - isType: kubernetes-cluster
 
+  # Group: Orchestrator
+  - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-orchestrator
+    disabled: true
+    pluginConfig:      
+      dynamicPlugins:
+        frontend:
+          janus-idp.backstage-plugin-orchestrator:
+            appIcons:
+              - name: orchestratorIcon
+                module: OrchestratorPlugin
+                importName: OrchestratorIcon
+            dynamicRoutes:
+              - path: /orchestrator
+                importName: OrchestratorPage
+                module: OrchestratorPlugin
+                menuItem:
+                  icon: orchestratorIcon
+                  text: Orchestrator                  
+  - package: ./dynamic-plugins/dist/backstage-plugin-orchestrator-backend-dynamic
```

Comment:
the path inside the image is `./dynamic-plugins/dist/janus-idp-backstage-plugin-orchestrator-backend-dynamic`